packages/x-charts/src/PieChart/PiePlot.tsx

```diff
@@ -1,48 +1,28 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import { SeriesContext } from '../context/SeriesContextProvider';
-import PieArc, { PieArcProps } from './PieArc';
-import PieArcLabel, { PieArcLabelProps } from './PieArcLabel';
 import { DrawingContext } from '../context/DrawingProvider';
 import {
-  DefaultizedPieValueType,
-  PieItemIdentifier,
-  PieSeriesType,
-} from '../models/seriesType/pie';
-
-const RATIO = 180 / Math.PI;
-
-function getItemLabel(
-  arcLabel: PieSeriesType['arcLabel'],
-  arcLabelMinAngle: number,
-  item: DefaultizedPieValueType,
-) {
-  if (!arcLabel) {
-    return null;
-  }
-  const angle = (item.endAngle - item.startAngle) * RATIO;
-  if (angle < arcLabelMinAngle) {
-    return null;
-  }
-
-  if (typeof arcLabel === 'string') {
-    return item[arcLabel]?.toString();
-  }
-
-  return arcLabel(item);
-}
+  PieArcPlot,
+  PieArcPlotProps,
+  PieArcPlotSlotComponentProps,
```

Comment:
Offtopic nitpick: Have you considered going with `PieArcPlotSlotProps` and `PieArcPlotSlots` naming instead, because `charts` do not have the deprecated `components` part? 🤔

Reply:
That's a good point. I did not thought about this naming convention :+1:

I will remove them in a distinct PR